homeassistant/components/freebox/device_tracker.py

```diff
@@ -58,7 +59,11 @@ def __init__(self, fbx):
         """Ensure the information from the Freebox router is up to date."""
         _LOGGER.debug("Checking Devices")
 
-        hosts = await self.connection.lan.get_hosts_list()
+        try:
+            hosts = await self.connection.lan.get_hosts_list()
+        except ClientConnectorError as err:
+            _LOGGER.error("Error getting freebox devices infos: %s", str(err))
```

Comment:
Please do not list the name of the integration in log messages, see [style guidelines](https://developers.home-assistant.io/docs/development_guidelines/):
```suggestion
            _LOGGER.error("Error getting info from devices: %s", err)
```